src/component/header/Header.js

```diff
@@ -1,10 +1,28 @@
 import React from "react";
 import "./header.css";
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
 import { useData } from "../../context/data-context";
+import { useAuth } from "../../context/auth-context";
 
 const Header = () => {
   const { state } = useData();
+  const { authState, authDispatch } = useAuth();
+
+  const navigate = useNavigate();
+  
+
+  const logoutHandler = () => {
+    navigate("/");
+    localStorage.removeItem("userAuthToken");
+    localStorage.removeItem("user");
+    authDispatch({ type: "LOGOUT" });
+  };
+
+  const loginBttonHandler = () => {
```

Comment:
For logout, you are using `logoutHandler`. For login also use the same naming convention. If not, then use the complete `button` spelling

Reply:
Done the changes